app/src/index.js

```diff
@@ -125,8 +138,7 @@ function show_debug_screen(wasm,msg) {
 
 /// Main entry point. Loads WASM, initializes it, chooses the scene to run.
 async function main() {
-    let target = window.location.href.split('/')
-    target.splice(0,3)
+    let target = window.location.search.substr(1).split("=");
```

Comment:
This part `window.location.search.substr(1)` should be refactored to a variable telling what it is.